### PR TITLE
Auspice v2 doc updates

### PR DESCRIPTION
**Pull request by misjailcisin**
Copy edits of nextstrain.org docs focused specifically on integrating auspice v2 updates.

### CODE REVIEW

File: static-site/content/docs/02-bioinformatics/05-data-formats.md [hunk 1]
```diff
@@ -43,3 +43,7 @@ The filename _must_ end with `_meta.json` and have the same prefix as the tree J
 
 Currently this is only used by the flu builds, and generates the frequencies panel you can see at [nextstrain.org/flu](https://nextstrain.org/flu). [Here](http://data.nextstrain.org/flu_seasonal_h3n2_ha_2y_tip-frequencies.json) is an example of this file.
 
+## Input files for Auspice
+_Full information on file format requirements for Auspice and how to get your data into the correct format will be updated soon_
+
+If you are interested in visualizing Newick trees, which is possible with CSV/TSV metadata files, then you may want to use [auspice.us](https://auspice-us.herokuapp.com/), which lets you drag and frop your files into the browser.
```
Comment (jameshadfield) [inline static-site/content/docs/02-bioinformatics/05-data-formats.md]: If you'd like to promote auspice.us then please add a caveat that it's in a very early prototype phase at the moment. 

P.S. `/frop/drop/`

File: static-site/content/docs/01-getting-started/04-local-installation.md [hunk 2]
```diff
@@ -13,11 +13,12 @@ If you have any issues with installing Augur/Auspice using any of these methods,
...
 #### Table of Contents:
-* [Install Augur & Auspice with conda (recommended)](#install-augur--auspice-with-conda-recommended)
-* [Install Augur with Python](#install-augur-with-python)
-* [Install Auspice using npm](#install-auspice-using-npm)
-* [Install Augur from source](#install-augur-from-source)
-* [Install Auspice from source](#install-auspice-from-source)
+- [Install Augur & Auspice with Conda (recommended)](#install-augur--auspice-with-conda-recommended)
+    - [Updating `Augur`, `Auspice`, and `Nextstrain`:](#updating-augur-auspice-and-nextstrain)
+- [Install Augur with Python](#install-augur-with-python)
+  - [Install Auspice using npm](#install-auspice-using-npm)
```
[the remark this answers is not in the document]
Comment (misjailcisin) [reply static-site/content/docs/01-getting-started/04-local-installation.md]: Fixed this in auspice-v2-rebased - not sure why it happened... some odd auto-format on my VS Code. Thanks for catching it!

File: static-site/content/docs/02-bioinformatics/05-data-formats.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Augur produces these JSONs via the `augur export` command -- see the [Augur docs
 We also define schemas for these JSONs (see below) and provide a validation tool to check JSONs against these schemas, `auspice validate` --  see the [Augur docs for more information [external link]](https://nextstrain-augur.readthedocs.io/en/stable/cli.html#validate).
 
 
-Auspice (version 1.x) currently requires two JSON files, with a third optional JSON.
+Auspice (version 2) currently requires two JSON files, with a third optional JSON.
```
[the remark this answers is not in the document]
Comment (misjailcisin) [reply static-site/content/docs/02-bioinformatics/05-data-formats.md]: I added the v1 language back in and pushed that commit. Can you link me to that table? I'm having trouble finding it.

[the remark this answers is not in the document]
Comment (jameshadfield) [reply static-site/content/docs/02-bioinformatics/05-data-formats.md]: https://github.com/nextstrain/auspice/blob/v2/docs-src/docs/releases/v2.md#new-dataset-json-format

File: static-site/content/docs/02-bioinformatics/05-data-formats.md (hunk 1 above)
Comment (misjailcisin) [reply static-site/content/docs/02-bioinformatics/05-data-formats.md]: Added this as it was something that was in the auspice v2 docs that I thought might be useful to have in the nextstrain.org docs. Happy to remove it especially if it's still very much a prototype. Pushed a commit with that language removed.

File: static-site/content/docs/02-bioinformatics/05-data-formats.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (misjailcisin) [reply static-site/content/docs/02-bioinformatics/05-data-formats.md]: Thank you! I just pushed a commit w/ the addition of the table.

**Comment by jameshadfield** [comment]
Closing as these changes are now in the auspice-v2-rebased branch.

**Pull request closed without merging**